packages/expo-blob/src/ExpoBlob.types.ts

```diff
@@ -0,0 +1,80 @@
+import { NativeModule, SharedObject } from 'expo';
+
+export declare class Blob {
+  /**
+   * The size of the Blob in bytes.
+   */
+  readonly size: number;
+  /**
+   * The MIME type of the Blob, or the empty string if the type cannot be determined.
+   */
+  readonly type: string;
+
+  /**
+   * Creates a new Blob object from the given parts and options.
+   * @param blobParts An array of BlobPart to include in the Blob.
+   * @param options An optional BlobPropertyBag dictionary.
+   */
+  constructor(blobParts?: BlobPart[], options?: BlobPropertyBag);
+
+  /**
+   * Returns a new Blob object containing the data in the specified range of bytes of the source Blob.
+   * @param start The starting byte index (inclusive) represented as a signed 32 bit integer (up to 2^31 - 1).
+   * @param end The ending byte index (exclusive) represented as a signed 32 bit integer (up to 2^31 - 1).
+   * @param contentType The MIME type of the new Blob. If not provided, defaults to an empty string.
+   * @returns A new Blob object containing the specified bytes.
```

Comment:
```suggestion
   * @param start The starting byte index (inclusive) represented as a signed 32 bit integer (up to 2^31 - 1).
   * @param end The ending byte index (exclusive) represented as a signed 32 bit integer (up to 2^31 - 1).
   * @param contentType The MIME type of the new Blob. If not provided, defaults to an empty string.
   * @returns A new Blob object containing the data in the specified range of bytes of the source Blob.
```

Let's avoid duplicating the return statements here and rest of this doc.